doc/redirects.json

```diff
@@ -3772,10 +3779,12 @@
   "what-if-your-favourite-vim-plugin-isnt-already-packaged": [
     "index.html#what-if-your-favourite-vim-plugin-isnt-already-packaged"
   ],
-  "vim-plugin-specificities": [
-    "index.html#vim-plugin-specificities"
+  "neovim-plugin-specificities": [
+    "index.html#neovim-plugin-specificities",
+    "neovim-plugin-specificities#vim-plugin-specificities"
```

Comment:
This should probably have been `index.html#vim-plugin-specificities`?

Reply:
Thanks for the feedback! And sorry that the experience was frustrating. Good point about input validation! The format is [documented in the source code for the tool dealing with it](https://github.com/nixos/nixpkgs/tree/master/pkgs/by-name/ni/nixos-render-docs/#design-considerations), but it sounds reasonable to indeed document it in place. `rename` and `move` are different because the ultimate goal is to be able to change the page to where content is rendered, which is not implemented yet.